lib/langchain/llm/openai.rb

```diff
@@ -40,6 +40,16 @@ def initialize(api_key:, llm_options: {}, default_options: {})
       @client = ::OpenAI::Client.new(access_token: api_key, **llm_options)
 
       @defaults = DEFAULTS.merge(default_options)
+      chat_parameters.update(
+        model: {default: "gpt-3.5-turbo"},
```

Comment:
Should we be using `@defaults[:chat_completion_model_name]` here?

Reply:
Yeah, the idea is just to provide some sensible defaults to allow people to get decent results quickly without thinking much.